test/data_processing/test_discriminator.py

```diff
@@ -80,3 +81,21 @@ def check_lda(lda1, lda2):
             return True
 
         self.assertRoundTripSerializable(lda, check_lda)
+
+    @requires_sklearn
+    def test_skclf_from_config(self):

```

Comment:
Can you add a test to see if the SkCLF can be serialized and deserialized?

Reply:
added in [df0b916](https://github.com/Qiskit/qiskit-experiments/pull/936/commits/df0b916978004c1babab7560b481a7dddf3ffade) but changed the discriminator to a `QuadraticDiscriminantAnalysis`.